src/pages/index.js

```diff
@@ -75,6 +71,17 @@ const IndexPage = () => (
         data-show-count="false">
     特設ページを見る</Link>
 </div>
+<div className={styles.textCenter}>
+  <h2>
+  Shop
+  </h2>
+  <p className={styles.specialDesc}>
+  SUZURIでGathering2022と10周年記念のグッズを販売中！<br />
+  </p>
+  <Link to="https://suzuri.jp/railsgirls-jp" target="_blank" className={styles.primaryButtonLarge}
+  data-show-count="false">
+  SUZURIを見る</Link> <br />
```

Comment:
[imo&nits]

SUZURIって言ってくれてうれしいですが、ここは「グッズを見る」とか「グッズをさがす」とかでもよさそうです ☺️ 
```suggestion
  グッズを見る</Link> <br />
```

Reply:
グッズを見るにしました〜！